code/modules/food_and_drinks/machinery/microwave.dm

```diff
@@ -64,6 +64,11 @@
 /obj/machinery/microwave/Exited(atom/movable/gone, direction)
 	if(gone in ingredients)
 		ingredients -= gone
+		if(!QDELING(gone) && ingredients.len && isitem(gone))
```

Comment:
why ingredients.len?

Reply:
So if you only cook one item in the microwave, it won't have the offset when it comes out. Can cut that if desired, just seemed like a nice touch.